frigidaire/__init__.py

```diff
@@ -209,9 +209,14 @@ def set_humidity(cls, humidity: int) -> List[Component]:
 
     @classmethod
     def set_temperature(cls, temperature: int) -> List[Component]:
-        # This is a restriction set by Frigidaire
-        if temperature < 60 or temperature > 90:
-            raise FrigidaireException("Temperature must be between 60 and 90 degrees, inclusive")
+        # These are restrictions set by Frigidaire
+        if ContainerId.UNIT == 0:
+            if temperature < 60 or temperature > 90:
+                raise FrigidaireException("Temperature must be between 60 and 90 degrees, inclusive")
+        
+        if ContainerId.UNIT == 1:
```

Comment:
I can't tell if `ContainerId.UNIT` will give us C or F, and what the value is exactly. Looking for guidance here @bm1549